MapboxNavigation/Constants.swift

```diff
@@ -1,6 +1,35 @@
 import Foundation
+import MapboxDirections
+
+typealias CongestionSegment = ([CLLocationCoordinate2D], CongestionLevel)
+
+/**
+ Line width base values used at zoom levels.
+ */
+public let lineWidthAtZoomLevels: [Int: MGLStyleValue<NSNumber>] = [
```

Comment:
Avoid polluting the global namespace in Objective-C: rename these public methods with the `MB` class prefix.

Reply:
Also, bring back the words “route line”, because these line widths are only applicable to route line styling.